bundle/tests/scorecard/kuttl/runtime_pullpolicy/03-default.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: example-runtime-component
+spec:
+  applicationImage: navidsh/demo-day
+  replicas: 1
```

Comment:
This test step fails. The new spec is merged with the previous spec, so you need to explicitly remove the configured pullPolicy. I think you can either add
```pullPolicy: ```
or
```pullPolicy: null```
to the end of this test file